### DISCUSSION TITLE
Лента ws2812b

### DISCUSSION THREAD

**Discussion by Vaqtincha** [DmytroKorniienko/FireLamp_EmbUI#118] [Q&A]
Проект можно использовать для ленты? (Те 120х1) эффекты как будут выглядеть? Интересует цветомузыка на esp8266 (вместо ардуинки) и удобное управление.

Полностью поддерживаю вашу позицию и страну. Я в качестве из иностранного языка знаю только русский (хотя сам с другой страны)

**Comment by DmytroKorniienko** [comment]
I understand some communication issues related with my choice not using russian language for discussion, so I can use English for simplify translation.

What about your questions:
Yes, you can use this project with a led strip instead matrix, but this possibility not fully tested, most of effects will work correctly but can looks like strange :)

Anyway you can test each effect and then prepare custom list with a most interesting ones. Please keep at mind - not all effects has sound reaction mode.

And if you want for this project possible make firmware for your native language with one exception - russian not supported at this moment. Thank you for understanding.

As alternative you can try wled project which developed for strips and some wled forks has sound reaction behavior as well.
